### PR TITLE
textures sample project flags are not set

### PR DESCRIPTION
**Pull request by pkevin-disguise**
Flags were set for index 0 all 3 times which resulted in undefined flags for index 1 and 2.

### CODE REVIEW

File: src/samples/Schema/Schema.cpp [hunk 1]
```diff
@@ -63,14 +63,14 @@ int mainImpl(int argc, char** argv)
     }
     
     std::vector<const char*> channels;
-    names.pushback("Default");
```
Comment (pkevin-disguise) [inline src/samples/Schema/Schema.cpp:66]: names does not exist, I've looked in the history and it was renamed to channels.

File: src/samples/Textures/Textures.cpp [hunk 2]
```diff
@@ -277,7 +277,7 @@ int mainImpl(int argc, char** argv)
     scoped.schema.scenes.scenes[0].parameters[1].options = nullptr;
     scoped.schema.scenes.scenes[0].parameters[1].dmxOffset = -1; // Auto
     scoped.schema.scenes.scenes[0].parameters[1].dmxType = RS_DMX_16_BE;
-    scoped.schema.scenes.scenes[0].parameters[0].flags = REMOTEPARAMETER_NO_SEQUENCE;
```
Comment (pkevin-disguise) [inline src/samples/Textures/Textures.cpp:280]: Flags was set for parameters index 0, but it should be set for index 1 here.
On line 291, it should be set for index 2.

**Review by dev-joss — APPROVED** [review]
LGTM 👍

**Comment by josh-disguise** [comment]
lgtm

**Pull request merged**